src/robot/api/deco.py

```diff
@@ -96,9 +128,38 @@ def decorator(func):
     return decorator
 
 
+C = TypeVar('C')
+
+
+# Decorator used without arguments, i.e. @library()
+@overload
+def library(scope: Type[C]) -> Type[C]:
```

Comment:
- `Type[C]` -> `'type[C]'`. This should be done also elsewhere.
- `scope: ...` -> `cls: ..., /`
- Remove unnecessary comment.